AngularJS/blogs.md

```diff
@@ -0,0 +1,3 @@
+Name | Link
+------------ | -------------
+A better way to learn AngularJS | thinkster.io/a-better-way-to-learn-angularjs
```

Comment:
Sir is there a way to open the link in new tab rather than same tab like adding target=_blank